cli.js

```diff
@@ -88,20 +93,26 @@ const onFeedMessage = (buf) => {
 		throw new Error('invalid feed: missing entity[]')
 	}
 
-	if (printAsJSON) {
-		process.stdout.write('[\n')
-	}
-	for (var i = 0; i < data.entity.length; ++i) {
-		const entity = data.entity[i];
-		const msg = printAsNDJSON || printAsJSON
-			? JSON.stringify(entity)
-			: inspect(entity, {depth, colors: printWithColors})
-		const isLastEntity = i == data.entity.length - 1;
-		const delimeter = (printAsJSON && !isLastEntity) ? ',\n' : '\n'
-		process.stdout.write(msg + delimeter)
-	}
-	if (printAsJSON) {
-		process.stdout.write(']\n')
+	const inspectOptions = {depth, colors: printWithColors};
+	if (includeAll) {
+		const msg = printAsJSON ? JSON.stringify(data) : inspect(data, inspectOptions);
+		process.stdout.write(msg);
+	} else {
```

Comment:
I think I would prefer to `return` early here, as IMHO it makes the code slightly more readable.